src/cli/global/install.rs

```diff
@@ -116,7 +119,23 @@ pub async fn execute(args: Args) -> miette::Result<()> {
         }
         last_updated_project = project;
     }
-    state_changes.report();
+    // When the environment is already up to date, we want to show a more detailed view about it.
+    if !state_changes.has_changed() {
```

Comment:
If certain environments contain no changes, but others do we should also print `environments up-to-date`.

Like here I would have expected, `Environment rattler-build is already up to date
![image](https://github.com/user-attachments/assets/c05ab727-b850-414e-a2bd-76c00b0f3a1a)